ui/components/ui/currency-display/currency-display.component.js

```diff
@@ -18,6 +25,10 @@ export default function CurrencyDisplay({
   denomination,
   currency,
   suffix,
+  prefixComponentWrapperProps = {},
+  textProps = {},
+  suffixProps = {},
+  boxProps = {},
```

Comment:
Excellent use of inversion of control here by allowing access to these components 💯